config/settings.yml

```diff
@@ -600,7 +600,7 @@ vba_documents:
     default_alert_email: ""
     in_flight_notification_hung_time_in_days: 14
     renotification_in_minutes: 240
-    update_stalled_notification_in_minutes: 180
+    update_stalled_notification_in_minutes: 5
```

Comment:
This setting is applied in the [vsp-infra-application-manifests repo](https://github.com/department-of-veterans-affairs/vsp-infra-application-manifests), so even though the value was changed here, the environment-specific settings in that repo will override it.

I suspect the 5 minute threshold may be too low and that we'll get a number of false positive alerts. 

Potential false positive scenarios: 
1. The job queue could be busy and therefore there could be a delay in the job being picked up.
2. `UploadProcessor` sometimes takes a minute or two to complete on large uploads.
3. The job could be orphaned if it takes more than a couple of minutes to complete and the pods are cycling, requiring it to be picked up by the next `VBADocuments::RunUnsuccessfulSubmissions` job. 

Each of the scenarios above will self-correct and there isn't really anything we can do to intervene, so I don't think an alert is helpful in those cases. What is helpful in my opinion is an alert when there are hundreds of records in "uploaded" status for ten minutes or so. That would indicate a problem that we would need to intervene on.

In lieu of the above suggestion, which would take more time to implement (and I would suggest we replace the `VBADocuments::SlackNotifier` job entirely if we went that route), I would recommend that we make the notification threshold around half of the current value, maybe 80 or 90 minutes, and schedule `VBADocuments::RunUnsuccessfulSubmissions` to run once per hour instead of once every two hours.

Reply:
I renamed the previous SlackNotifier to SlackInflightNotifier and removed reporting on stuck uploaded BI submissions.  Now SlackInflightNotifier only reports on uploads stuck in central mail and the weird invalid_parts_alert(which i really don't think we see hits on in production)

I created a ticket to consider either moving the stuck inflghts to the new SlackStatusNotifier or just getting rid of them: https://jira.devops.va.gov/browse/API-35575


The new SlackStatusNotifer is now reporting on suck uploaded BI Submissions.  
I removed reporting